correlation_context/HTTP_HEADER_FORMAT.md

```diff
@@ -44,6 +43,13 @@ OWS         = <Optional white space, as defined in [[RFC7230], Section 3.2.3.](h
 ## Example
 `key1=value1[;property1;property2], key2 = value2[;property3]`
 
+### list
+List of key-value pairs with optional properties attached.
+It can not be guaranteed that keys are unique.
+
+Consumers MUST be able to handle duplicate keys while producers SHOULD try to dedupe the list.
```

Comment:
```suggestion
Consumers MUST be able to handle duplicate keys while producers SHOULD try to deduplicate the list.
```

Reply:
I think we should prescribe how deduping should be done, e.g. last entry wins